iroh-rpc-client/Cargo.toml

```diff
@@ -9,6 +9,7 @@ description = "RPC type client for iroh"
 
 [dependencies]
 async-stream = "0.3.0"
+crossterm = "0.23.2"
```

Comment:
I would like to keep the cli specific stuff in iroh-ctl and have the client only give structured data, as this will be used in different environments